packaging/windows/clisdk/bundle.thm

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think these bundle and dotnet files are for the .NET Core SDK MSI, and shouldn't be needed in this repo.